recipes/cmakelang/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "cmakelang" %}
+{% set version = "0.6.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 03982e87b00654d024d73ef972d9d9bb0e5726cdb6b8a424a15661fb6278e67f
+  patches:
+    - patch/0001-Fix-setup.py.patch
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - cmake-annotate = cmakelang.annotate:main
+    - cmake-format = cmakelang.format.__main__:main
+    - cmake-lint = cmakelang.lint.__main__:main
+    - cmake-genparsers = cmakelang.genparsers:main
+    - ctest-to = cmakelang.ctest_to:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - six >=1.13.0
+    - pyyaml >=5.3
+    - jinja2 >=2.10.3
+
+test:
+  imports:
+    - cmakelang
+    - cmakelang.format
+    - cmakelang.lex
+    - cmakelang.lint
+    - cmakelang.parse
+    - cmakelang.parse.funs
+  commands:
+    - cmake-format -h
+    - cmake-lint -h
+    - cmake-annotate -h
+    - cmake-genparsers -h
+    - ctest-to -h
+
```

To ensure the resulting Python environment is consistent

```suggestion
    - pip check
  requires:
    - pip

```